package/yast2-network.changes

```diff
@@ -1,6 +1,13 @@
 -------------------------------------------------------------------
 Wed Jan 14 11:26:43 UTC 2015 - mfilka@suse.com
 
+- fate#315162
+  - IPoIB mode configuration in YaST
+- 2.17.202
+
+-------------------------------------------------------------------
+Thu Jan  8 21:07:28 UTC 2015 - mfilka@suse.com
```

Comment:
Changing the timestamp of an existing entry would result in a failed check "changelog is not monotonic", but here we are lucky because the previous change is not accepted yet :)